tesseract_motion_planners/descartes/src/descartes_collision.cpp

```diff
@@ -45,10 +45,9 @@ DescartesCollision::DescartesCollision(const tesseract_environment::Environment&
   , debug_(debug)
 {
   contact_manager_->setActiveCollisionObjects(active_link_names_);
-  contact_manager_->setCollisionMarginData(collision_check_config_.collision_margin_data,
-                                           collision_check_config_.collision_margin_override_type);
   contact_manager_->setIsContactAllowedFn(
       [this](const std::string& a, const std::string& b) { return isContactAllowed(a, b); });
+  contact_manager_->applyContactManagerConfig(collision_check_config.contact_manager_config);
```

Comment:
Same here.

Reply:
The local acm and method is removed and now leverage the parameter in the config